README.md

```diff
@@ -17,11 +32,449 @@ Xcode 16 or later.
+
+```swift
+let messagesSubscription = try await room.messages.subscribe(bufferingPolicy: .unbounded)
+for await message in messagesSubscription {
+    print("Message received: \(message)")
+}
+```
+
+To stop listening for the new messages, call the `finish` method on the returned subscription instance.
+
+### Sending messages
+
+To send a message, simply call `send` on the room `messages` property, with the message you want to send:
+
+```swift
+let message = try await room.messages.send(params: .init(text: "hello"))
+```
+
+### Retrieving message history
+
+The messages object also exposes the `get` method which can be used to request historical messages in the chat room according
+to the given criteria. It returns a paginated response that can be used to request more messages:
+
+```swift
+let paginatedResult = try await room.messages.get(options: .init(orderBy: .newestFirst))
+print(paginatedResult.items)
+
+if paginatedResult.hasNext {
+    let next = try await paginatedResult.next!
+    print(next.items)
+} else {
+    print("End of messages")
+}
+```
+
+### Retrieving message history for a subscribed listener
+
+In addition to being able to unsubscribe from messages, the return value from `messages.subscribe` also includes the `getPreviousMessages`
+method. It can be used to request historical messages in the chat room that were sent up to the point that a particular listener was subscribed. It returns a
+paginated response that can be used to request for more messages:
+
+```swift
+let messagesSubscription = try await room.messages.subscribe(bufferingPolicy: .unbounded)
+let paginatedResult = try await messagesSubscription.getPreviousMessages(params: .init(limit: 50)) // `orderBy` here is ignored and always `newestFirst`
+print(paginatedResult.items)
+
+if paginatedResult.hasNext {
+    let next = try await paginatedResult.next!
+    print(next.items)
+} else {
+    print("End of messages")
+}
+```
+
+## Online status
+
+### Retrieving online members
+
+You can get the complete list of currently online or present members, their state and data, by calling the `presence#get` method which returns
+a list of the presence messages, where each message contains the most recent data for a member:
+
+```swift
+// Retrieve all users entered into presence as an array:
+let presentMembers = try await room.presence.get()
+
+// Retrieve the status of specific users by their clientId:
+let presentMember = try await room.presence.get(params: .init(clientID: "clemons123"))
+
+// To check whether the user is online or not:
+let isPresent = try await room.presence.isUserPresent(clientID: "clemons123")
+```
+
+### Entering the presence set
+
+To appear online for other users, you can enter the presence set of a chat room. While entering presence, you can provide optional data that
+will be associated with the presence message:
+
+```swift
+try await room.presence.enter(data: .init(userCustomData: ["status": .string("Online")]))
```

Comment:
These can be updated now that #189 is merged

Reply:
done https://github.com/ably/ably-chat-swift/compare/49360b63fe22c18f2a37a5e651adcb9ea86cf905..6472e96566e4bb8a82ad6b336fbc7718a0a14557